src/libraries/Common/src/System/Globalization/FormatProvider.Number.cs

```diff
@@ -607,7 +607,7 @@ internal static unsafe bool TryStringToNumber(ReadOnlySpan<char> str, NumberStyl
                 "($#)", "-$#", "$-#", "$#-",
                 "(#$)", "-#$", "#-$", "#$-",
                 "-# $", "-$ #", "# $-", "$ #-",
-                "$ -#", "#- $", "($ #)", "(# $)"
+                "$ -#", "#- $", "($ #)", "(# $)", "$- #"
```

Comment:
Nit: All lines above are 4 strings per line. Follow the style a put it on a new line?